src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,198 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static final String INDENT = "      ";
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        printWelcome();
+
+        boolean isBye = false;
+        while (!isBye) {
+            String userInput = readUserInput();
+            String command = getCommand(userInput);
+            String commandParameter = getCommandParameter(userInput);
+
+            System.out.println(LINE);
+
+            try {
+                switch(command) {
+                case "bye":
+                    isBye = true;
+                    break;
+                case "todo":
+                    addTodo(commandParameter);
+                    break;
+                case "deadline":
+                    addDeadline(commandParameter);
+                    break;
+                case "event":
+                    addEvent(commandParameter);
+                    break;
+                case "done":
+                    markTaskAsDone(commandParameter);
+                    break;
+                case "list":
+                    listTasks();
+                    break;
+                default:
+                    printError("I'm sorry, I don't know what that means.");
+                }
+            } catch (DukeException e) {
+                printError(e.getMessage());
+            }
+        }
+
+        printGoodbye();
+    }
```

Comment:
Perhaps transferring the parsing code to a Parser class would be a good idea to shorten the Main() method?

![image](https://user-images.githubusercontent.com/44335297/92136970-3eab8580-ee3f-11ea-9407-5730e19dc973.png)